docs/design/sql/18-data-links.md

```diff
@@ -676,7 +654,6 @@ these privileges related to data links:
 
 * new SQL actions `create-datalink` and `drop-datalink`. To CREATE OR REPLACE a
   data link, one needs permission to both actions
-
 * new SQL action: `view-datalink`. Without this permission the user won’t be
```

Comment:
how do we protect against:
```
CREATE or replace MAPPING sqlCatalog
EXTERNAL NAME "__sql.catalog"
TYPE IMap
OPTIONS (
'keyFormat'='java',
'keyJavaClass' = 'java.lang.String',
'valueFormat' = 'java',
'valueJavaClass' = 'java.lang.Object'
)

SELECT * FROM sqlCatalog
```
?

Reply:
ok: "access to this map must be denied". Do I understand correctly, that this is not possible in OS? Will it be default in EE?

Reply:
Yes, check https://github.com/hazelcast/hazelcast-enterprise/pull/5785

Reply:
I think that that PR should be linked as EE part of this one

Reply:
It's a **separate** PR with **separate** ticket

Reply:
fine. but it tests changes added in this PR